internal/cmd/connector.go

```diff
@@ -68,6 +68,11 @@ func newConnectorCmd() *cobra.Command {
 	cmd.Flags().String("ca-cert", "", "Path to CA certificate file")
 	cmd.Flags().String("ca-key", "", "Path to CA key file")
 	cmd.Flags().Bool("server-skip-tls-verify", false, "Skip verifying server TLS certificates")
+	cmd.Flags().String("endpoint-addr-host", "", "Connector proxy host")
+	cmd.Flags().String("endpoint-addr-port", "", "Connector proxy port")
+	cmd.Flags().String("addr-http", "", "HTTP address to bind to")
+	cmd.Flags().String("addr-https", "", "HTTPS address to bind to")
+	cmd.Flags().String("addr-metrics", "", "HTTP address to bind to for metrics")
```

Comment:
What are these extra command line options for?

Reply:
Might be nice to avoid these, especially if we're looking at remove the `addr-` ones with the proxy.

We can use the equivalent env vars if we want a way to set these without the config file.

Reply:
Sounds good. I'll omit them for now.